todo/src/components/Todo.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { add_todo, update_completed, delete_item } from '../actions/index.js';
+
+import './todo.css';
+
+class ToDo extends React.Component {
+  state = {
+    todo: '',
+  };
+
+  render() {
+    return (
+      <div>
+        <h1 className="header"> To Do List </h1>
+        <div className="list">
+          <ul style={{ width: '100%' }}>
+            {this.props.todos.map((item, index) => {
+              return (
+                <div
+                  style={{
+                    marginBottom: '5px',
+                    display: 'flex',
+                    flexFlow: 'row',
+                    padding: '5px',
+                    border: '2px solid blue',
+                  }}
+                  key={item.id}>
+                  <button
+                    style={{ marginLeft: '5px' }}
+                    onClick={() => this.props.delete_item(index)}>
+                    delete
+                  </button>
+                  <li style={{ margin: '5px' }} onClick={() => this.props.update_completed(item)}>
+                    {item.todo}
+                  </li>
+                </div>
+              );
```

Comment:
I would like to see this split out onto a separate view.  Theres a lot going on in this one and it would be easier to follow along if it was split out more